src/components/SearchBar.js

```diff
@@ -0,0 +1,27 @@
+import { useState } from "react";
+import RecipeGallery from "./RecipeGallery";
+
+function SearchBar() {
+
+    const [userInput, setUserInput] = useState("");
+
+
+    const handleInput = (event) => {
```

Comment:
I love your idea of creating the ability for a user to like a recipe. I feel like some of our past code-alongs could be helpful with implementing this! Also, I think it would be very doable to send the "liked" recipes up to firebase. Maybe creating a separate page or option to "view liked recipes" and then display liked recipes stored in firebase on the page could be cool!